src/browser_launchers/playwright_launcher.js

```diff
@@ -74,17 +18,38 @@ export function apifyOptionsToLaunchOptions(launchContext) {
  *   Example: `http://bob:pass123@proxy.example.com:1234`.
  * @property {boolean} [useChrome=false]
  *   If `true` and `executablePath` is not set,
- *   Puppeteer will launch full Google Chrome browser available on the machine
+ *   Playwright will launch full Google Chrome browser available on the machine
  *   rather than the bundled Chromium. The path to Chrome executable
  *   is taken from the `APIFY_CHROME_EXECUTABLE_PATH` environment variable if provided,
  *   or defaults to the typical Google Chrome executable location specific for the operating system.
  *   By default, this option is `false`.
- * @property {(string|Object)} [launcher]
+ * @property {Object} [launcher]
  *   By default this function uses require("playwright").chromium`.
  *   If you want to use a different browser you can pass it by this property as `require("playwright").firefox
  */
 
-/**
+export class PlaywrightLauncher extends BrowserLauncher {
+    static optionsShape = {
+        ...BrowserLauncher.optionsShape,
+        launcher: ow.optional.object,
+    }
+
+    /**
+    * @param {PlaywrightLaunchContext} launchContext
+    * All `PlaywrightLauncher` parameters are passed via an launchContext object.
```

Comment:
param and text flipped

Reply:
also typo in `a launchContext`

Reply:
What do you mean fliped?  The text should be a description of the launchContext.